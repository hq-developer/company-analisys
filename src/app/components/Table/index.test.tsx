import React from 'react';
import { render } from '@testing-library/react';

import Component from './index';

describe('<TableComponent/>', () => {
  it('should render the component', () => {
    const { container } = render(
        <Component
            headers={["Col1", "Col2", "Col3"]}
            dataSource={[
                [1, 2, 3],
                ['one', 'two', 'three'],
            ]}
        />
    );
    expect(container).toMatchSnapshot();
  });
    
});