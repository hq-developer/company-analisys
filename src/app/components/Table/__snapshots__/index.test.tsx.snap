// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableComponent/> should render the component 1`] = `
.c0 {
  width: 100%;
}

<div>
  <table
    class="c0"
  >
    <thead>
      <tr
        class=""
      >
        <td
          class=""
        >
          Col1
        </td>
        <td
          class=""
        >
          Col2
        </td>
        <td
          class=""
        >
          Col3
        </td>
      </tr>
    </thead>
    <tbody>
      <tr
        class=""
      >
        <td
          class=""
        >
          1
        </td>
        <td
          class=""
        >
          2
        </td>
        <td
          class=""
        >
          3
        </td>
      </tr>
      <tr
        class=""
      >
        <td
          class=""
        >
          one
        </td>
        <td
          class=""
        >
          two
        </td>
        <td
          class=""
        >
          three
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
